cheroot/_compat.py

```diff
@@ -8,6 +8,12 @@
 
 import six
 
+# Not used in this file, imported elsewhere as 'from ._compat import selectors'
+try:
+    import selectors
+except ImportError:
+    import selectors2 as selectors  # noqa: F401
```

Comment:
The top comment is redundant: the whole point of this module is to have shims like this.
```suggestion
try:
    import selectors  # lgtm [py/unused-import]
except ImportError:
    import selectors2 as selectors  # noqa: F401  # lgtm [py/unused-import]
```